pallets/automation-time/src/benchmarking.rs

```diff
@@ -189,8 +189,17 @@ benchmarks! {
 	schedule_xcmp_task_full {
 		let v in 1..T::MaxExecutionTimes::get();
 
+		// TODO: Additional read/writes accounts for AssetRegistry assets unavailable.
```

Comment:
modifying the weight here means no need to manually update github action weights manually.